js/index.js

```diff
@@ -203,19 +216,11 @@ var FacebookInstant = Class(function () {
         if (opts.no_message) {
           return;
         }
-        this.sendMessage({
-          data: {
-            id: Date.now() + '_' + player.getID(),
-            type: 'invite',
-            player_id: player.getID(),
-            name: player.getName(),
-            action: opts.action
-          },
-          text: opts.message,
-          image: opts.image,
-          template: opts.template || 'invite'
-        });
-      })); 
+        this.sendMessage(send_data);
+      }));
+    } else {
+      this.sendMessage(send_data);
+    }
```

Comment:
we can remove the else condition and move the sendMessage outside ?

Reply:
the first sendMessage comes in a then function. the sendMessage in else is send immediately

Reply:
understood. makes sense now